### DISCUSSION TITLE
multiple servers

### DISCUSSION THREAD

**Discussion by DmiRials** [rustdesk/rustdesk#1488] [General]
How can I make all clients contact 2 rendezvous servers and use 2 relay servers?
this is necessary for load balancing and fault tolerance

**Comment by elveos** [comment]
I could see this potentially working with FQDN having multiple DNS A records. Round robin check-until-succeeded would have to be implemented for this on both receiving machine and client end since receiving machine seems to stick to the old IP address until restarted (opened issue #2245 for this).

Also, noticed the line in `~/.config/rustdesk/RustDesk.toml`:
```
custom-rendezvous-server = 'remote.example.com'
rendezvous-servers = 'rs-ny.rustdesk.com,rs-sg.rustdesk.com,rs-cn.rustdesk.com'
```
Seems like some kind of load balanced setup is already being used for public server? 
I tested this by changing to `custom-rendezvous-server = 'remote.example.com,remote2.example.com'` (with `remote2` A record not actually existing), but that ended up getting stuck getting at 'connecting'.
